### ISSUE TITLE
Blank setup page. Ubuntu 18 and 20

### ISSUE THREAD

**Issue by d3lta105** [grokability/snipe-it#9656]
#### Please confirm you have done the following before posting your bug report:

- [X] I have enabled debug mode 
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

**Describe the bug**

**To Reproduce**
1. Go to the site ip address using any browser
2. Click on nothing
3. Scroll down nowehere
4. See blank webpage

**Expected behavior**
It's supposed to redirect to the setup page after fresh install. The URL does change to "http://192.168.x.x/setup" and the site icon changes to SnipeIT icon in the browser tab. But page is blank.

**Screenshots**
it's just a blank page

**Server (please complete the following information):**
 - Snipe-IT Version: latest
 - OS: tried on Ubuntu server 20 LTS, switched to 18, same issue
 - Web Server: Apache. 
 - PHP Version latest

**Desktop (please complete the following information):**
 - OS: ubuntu
 - Browser firefox
 - Version 88

**Additional context**
- Is this a fresh install or an upgrade?: Fresh using the script
- What OS and web server you're running Snipe-IT on Ubuntu Apache
- What method you used to install Snipe-IT (install.sh, manual installation, docker, etc) Install.sh
- Include what you've done so far in the installation, and if you got any error messages along the way. just got past generating the keys
- Indicate whether or not you've manually edited any data directly in the database: nope

**Comment by JakeMadsen** [comment]
Did you find a solution to this? I am currently also just getting a blank page

**Comment by cringerjs** [comment]
I have the same issue.  Fresh install of Ubuntu 20.04 and updated.  Run the install.sh command and when finished it says I can go to the site.  When I navigate to the site it redirects to /setup but the page is just blank.

I checked the Apache logs and it does generate errors:
PHP Fatal error:  Uncaught ErrorException: file_put_contents(/var/www/html/snipeit/storage/framework/cache/cf/90/cf906e7195218453e4ff6f1bdcdc6b2d9f6cae7

I ended up resetting the permissions on the storage directory and then it worked correctly.  Not sure why this wouldn't work correctly on a fresh install.
